2024-07-16 09:29:18,518 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:29:18,518 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:29:18,593 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:29:19,218 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:29:19,220 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:30:09,964 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:55]
2024-07-16 09:30:10,108 INFO: 127.0.0.1 - - [16/Jul/2024 09:30:10] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:30:10,302 INFO: 127.0.0.1 - - [16/Jul/2024 09:30:10] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:30:10,306 INFO: 127.0.0.1 - - [16/Jul/2024 09:30:10] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:30:10,376 INFO: 127.0.0.1 - - [16/Jul/2024 09:30:10] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:30:10,403 INFO: 127.0.0.1 - - [16/Jul/2024 09:30:10] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:32:03,166 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:32:03,174 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:32:03,175 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:32:04,016 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:57:06,132 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:57:06,133 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:57:06,151 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:57:06,847 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:57:06,849 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,478 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:55]
2024-07-16 09:58:16,559 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\flask\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,561 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\orm\\scoping.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,564 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\orm\\session.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,565 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\orm\\state_changes.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,566 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\util\\langhelpers.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,567 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\orm\\unitofwork.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,568 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\orm\\persistence.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,570 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\engine\\base.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,574 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\sql\\elements.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,576 INFO:  * Detected change in 'C:\\Users\\Administrator\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\sqlalchemy\\engine\\default.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,621 INFO: 127.0.0.1 - - [16/Jul/2024 09:58:16] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,718 INFO: 127.0.0.1 - - [16/Jul/2024 09:58:16] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,726 INFO: 127.0.0.1 - - [16/Jul/2024 09:58:16] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,755 INFO: 127.0.0.1 - - [16/Jul/2024 09:58:16] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:16,783 INFO: 127.0.0.1 - - [16/Jul/2024 09:58:16] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:17,699 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:18,445 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 09:58:18,448 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:00:44,852 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\_pydatetime.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:00:45,048 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:00:45,975 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:00:45,979 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:47,225 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:47,226 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:47,237 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:48,030 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:48,032 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:56,106 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:55]
2024-07-16 10:03:56,237 INFO: 127.0.0.1 - - [16/Jul/2024 10:03:56] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:56,349 INFO: 127.0.0.1 - - [16/Jul/2024 10:03:56] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:56,357 INFO: 127.0.0.1 - - [16/Jul/2024 10:03:56] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:56,379 INFO: 127.0.0.1 - - [16/Jul/2024 10:03:56] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:03:56,408 INFO: 127.0.0.1 - - [16/Jul/2024 10:03:56] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:04:03,347 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:55]
2024-07-16 10:04:03,411 INFO: 127.0.0.1 - - [16/Jul/2024 10:04:03] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:04:03,427 INFO: 127.0.0.1 - - [16/Jul/2024 10:04:03] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:04:03,435 INFO: 127.0.0.1 - - [16/Jul/2024 10:04:03] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:04:03,470 INFO: 127.0.0.1 - - [16/Jul/2024 10:04:03] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:04:03,501 INFO: 127.0.0.1 - - [16/Jul/2024 10:04:03] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:07:45,438 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:07:45,447 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:07:46,251 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:07:47,196 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:07:47,199 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:10,459 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:10,460 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:10,461 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:10,689 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:11,656 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:11,659 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:32,454 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:32,455 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:32,464 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:33,268 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:33,271 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:38,160 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:53]
2024-07-16 10:08:38,291 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:38] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:38,400 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:38] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:38,408 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:38] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:38,434 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:38] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:38,461 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:38] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:46,061 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:53]
2024-07-16 10:08:46,173 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:46] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:46,253 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:46] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:46,254 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:46] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:08:46,300 INFO: 127.0.0.1 - - [16/Jul/2024 10:08:46] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:17:29,546 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:17:29,547 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:17:29,563 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:17:30,233 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:17:30,235 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:18:32,005 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:55]
2024-07-16 10:18:32,062 INFO: 127.0.0.1 - - [16/Jul/2024 10:18:32] "[35m[1mGET / HTTP/1.1[0m" 500 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:18:32,157 INFO: 127.0.0.1 - - [16/Jul/2024 10:18:32] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:18:32,256 INFO: 127.0.0.1 - - [16/Jul/2024 10:18:32] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:18:32,615 INFO: 127.0.0.1 - - [16/Jul/2024 10:18:32] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:18:32,867 INFO: 127.0.0.1 - - [16/Jul/2024 10:18:32] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:55,725 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:55,726 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:55,726 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:56,595 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:57,312 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:22:57,319 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:08,879 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:08,879 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:08,897 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:09,384 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:09,390 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:26,035 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:23:26,038 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:49]
2024-07-16 10:23:26,077 INFO: 127.0.0.1 - - [16/Jul/2024 10:23:26] "GET / HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:26,079 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\unicode_escape.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:26,216 INFO: 127.0.0.1 - - [16/Jul/2024 10:23:26] "[33mGET /favicon.ico HTTP/1.1[0m" 404 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:26,607 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:27,213 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:27,222 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:23:29,466 INFO: 127.0.0.1 - - [16/Jul/2024 10:23:29] "GET /login HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:08,313 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:08] "GET /login HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:09,934 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:09] "GET /login HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:12,629 INFO: Página principal cargada [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:12,638 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:12] "GET / HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:14,355 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:14] "GET /login HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:46,820 INFO: Inicio de sesión exitoso: armoniaadmin [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:46,825 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:46] "[32mPOST /login HTTP/1.1[0m" 302 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:47,105 INFO: Dashboard cargado correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:47,126 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:47] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:47,506 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:47] "GET /static/css/styles.css HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:54,792 INFO: Dashboard cargado correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:54,794 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:54] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:55,099 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:55] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:56,586 INFO: Dashboard cargado correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:56,588 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:56] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:56,718 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:56] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:57,347 INFO: Listado de alumnos obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:26:57,365 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:57] "GET /admin/alumnos HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:26:57,649 INFO: 127.0.0.1 - - [16/Jul/2024 10:26:57] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:27:02,656 INFO: 127.0.0.1 - - [16/Jul/2024 10:27:02] "[33mGET /admin/configuracion HTTP/1.1[0m" 404 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:29:26,398 INFO: Listado de profesores obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:29:26,441 INFO: 127.0.0.1 - - [16/Jul/2024 10:29:26] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:29:26,467 INFO: 127.0.0.1 - - [16/Jul/2024 10:29:26] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:29:30,421 INFO: Listado de inscripciones obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:29:30,437 INFO: 127.0.0.1 - - [16/Jul/2024 10:29:30] "GET /admin/inscripciones HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:29:30,723 INFO: 127.0.0.1 - - [16/Jul/2024 10:29:30] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:12,687 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_database.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:12,688 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:12,690 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_gestion_de_permisos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:13,387 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:14,332 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:31:14,350 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:33:57,951 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_usuarios.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:33:58,969 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:34:00,970 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:34:00,978 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:10,434 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:10,434 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:10,888 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:11,334 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:11,338 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,104 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,104 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,113 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,427 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,830 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:42:17,834 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:43:08,751 INFO: Listado de descuentos obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:43:08,777 INFO: 127.0.0.1 - - [16/Jul/2024 10:43:08] "GET /admin/descuentos HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:43:08,901 INFO: 127.0.0.1 - - [16/Jul/2024 10:43:08] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:43:12,320 INFO: Listado de alumnos obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:43:12,325 INFO: 127.0.0.1 - - [16/Jul/2024 10:43:12] "GET /admin/alumnos HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:43:12,621 INFO: 127.0.0.1 - - [16/Jul/2024 10:43:12] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,867 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,868 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\aliases.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,869 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\utf_8.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,870 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\encodings\\cp1252.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,883 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\logging\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,884 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\re\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,885 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\enum.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,886 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\types.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,887 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\operator.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,888 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\functools.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,889 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\collections\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,890 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\keyword.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,891 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\reprlib.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,893 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\re\\_compiler.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,894 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\re\\_parser.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,895 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\re\\_constants.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,896 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\re\\_casefix.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,897 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\copyreg.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,897 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\traceback.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,898 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\collections\\abc.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,899 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\linecache.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,900 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\tokenize.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,901 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\token.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,902 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\textwrap.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,904 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\contextlib.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,905 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\warnings.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,906 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\weakref.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,907 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\_weakrefset.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,908 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\string.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:39,909 INFO:  * Detected change in 'C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\Lib\\threading.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:40,401 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:40,781 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:54:40,785 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:56:19,713 INFO: Listado de profesores obtenido correctamente [in C:\Users\Administrator\Desktop\F5\Grupo3_Musica\logs.py:56]
2024-07-16 10:56:19,731 INFO: 127.0.0.1 - - [16/Jul/2024 10:56:19] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 10:56:19,862 INFO: 127.0.0.1 - - [16/Jul/2024 10:56:19] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,033 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,034 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_database.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,035 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_gestion_de_permisos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,035 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,036 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_usuarios.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,036 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\logs.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,037 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,038 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,039 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:15,497 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:01:17,061 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:02:21,026 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:07:49,835 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:08:53,765 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:08:54,417 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:08:54,423 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,873 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,874 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_database.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,874 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_gestion_de_permisos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,875 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,875 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_usuarios.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,951 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,952 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:10,952 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:10:11,026 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:11:23,049 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:13:14,190 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:14:27,180 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:14:27,722 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:14:29,191 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:15:28,987 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:15:29,694 WARNING:  * Debugger is active! [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:15:29,703 INFO:  * Debugger PIN: 537-427-931 [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,335 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\__init__.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,336 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_database.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,336 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_gestion_de_permisos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,337 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_insertar_datos.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,337 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\control\\control_usuarios.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,411 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,411 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:38,412 INFO:  * Detected change in 'C:\\Users\\Administrator\\Desktop\\F5\\Grupo3_Musica\\app.py', reloading [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:16:39,236 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:17:02,178 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:17:03,635 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:17:54,318 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 11:17:55,876 INFO:  * Restarting with watchdog (windowsapi) [in C:\Users\Administrator\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:45,454 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:45,454 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:45,458 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:45,842 WARNING:  * Debugger is active! [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:45,848 INFO:  * Debugger PIN: 733-530-386 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:57,920 INFO: Página principal cargada [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:15:57,943 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:15:57,949 INFO: 127.0.0.1 - - [16/Jul/2024 12:15:57] "GET / HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:58,121 INFO: 127.0.0.1 - - [16/Jul/2024 12:15:58] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:15:58,122 INFO: 127.0.0.1 - - [16/Jul/2024 12:15:58] "[36mGET /static/images/logo.png HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:00,414 INFO: Página principal cargada [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:00,416 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:00,429 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:00] "GET / HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:00,450 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:00] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:00,451 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:00] "[36mGET /static/images/logo.png HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:01,991 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:01] "GET /login HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:02,012 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:02] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:24,628 INFO: Inicio de sesión exitoso: armoniaadmin [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:24,630 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:24,633 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:24] "[32mPOST /login HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:24,638 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:24,640 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:24,647 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:24] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:24,675 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:24] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:26,602 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:26,603 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:26,606 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:26] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:26,619 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:26] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:27,756 INFO: Listado de alumnos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:27,757 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:27,764 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:27] "GET /admin/alumnos HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:27,786 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:27] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:30,578 INFO: Listado de profesores obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:30,580 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:30,586 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:30] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:30,605 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:30] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:32,186 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:32,187 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:32,197 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:32] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:32,219 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:32] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:33,069 INFO: Listado de inscripciones obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:33,070 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:33,080 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:33] "GET /admin/inscripciones HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:33,104 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:33] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:36,138 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:36,139 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:36,141 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:36] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:36,159 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:36] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:37,183 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:37] "GET /admin/clases/nueva HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:37,201 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:37] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:46,421 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:46] "[32mPOST /admin/clases/nueva HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:46,425 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:46,426 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:46,428 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:46] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:46,452 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:46] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:50,922 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:50] "GET /admin/clases/18/editar HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:50,947 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:50] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:57,797 INFO: Clase editada correctamente: ID 18 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:57,798 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:57,799 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:57] "[32mPOST /admin/clases/18/editar HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:57,805 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:16:57,807 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:16:57,809 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:57] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:16:57,836 INFO: 127.0.0.1 - - [16/Jul/2024 12:16:57] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:11,519 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:11] "GET /admin/clases/18/editar HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:11,539 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:11] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:15,937 INFO: Clase editada correctamente: ID 18 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:17:15,938 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:17:15,939 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:15] "[32mPOST /admin/clases/18/editar HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:15,945 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:17:15,946 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:17:15,948 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:15] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:15,978 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:15] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:26,854 INFO: Clase eliminada correctamente: ID 18 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:17:26,856 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:17:26,857 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:26] "[32mPOST /admin/clases/18/eliminar HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:26,861 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:17:26,864 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:17:26,865 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:26] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:26,890 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:26] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:29,934 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:17:29,935 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:17:29,936 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:29] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:17:29,958 INFO: 127.0.0.1 - - [16/Jul/2024 12:17:29] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:49,714 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:49,715 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:49,719 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:50,062 WARNING:  * Debugger is active! [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:50,066 INFO:  * Debugger PIN: 733-530-386 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:54,932 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:54,936 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:54,941 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:54] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:55,007 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:55] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:56,778 INFO: Listado de profesores obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:56,779 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:56,782 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:56] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:56,817 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:56] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:57,527 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:57,529 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:57,533 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:57] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:57,555 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:57] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:58,007 INFO: Listado de inscripciones obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:58,008 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:58,014 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:58] "GET /admin/inscripciones HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:58,037 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:58] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:59,029 INFO: Listado de profesores obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:59,031 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:59,037 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:59] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:59,055 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:59] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:59,574 INFO: Listado de alumnos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:26:59,575 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:26:59,583 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:59] "GET /admin/alumnos HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:26:59,616 INFO: 127.0.0.1 - - [16/Jul/2024 12:26:59] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:11,088 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:11,088 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:11,091 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:11,436 WARNING:  * Debugger is active! [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:11,439 INFO:  * Debugger PIN: 733-530-386 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:23,219 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:23,219 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:23,222 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:23,557 WARNING:  * Debugger is active! [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:23,560 INFO:  * Debugger PIN: 733-530-386 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:41,795 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:29:41,799 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:29:41,810 INFO: 127.0.0.1 - - [16/Jul/2024 12:29:41] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:41,922 INFO: 127.0.0.1 - - [16/Jul/2024 12:29:41] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:43,682 INFO: Listado de profesores obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 12:29:43,684 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 12:29:43,690 INFO: 127.0.0.1 - - [16/Jul/2024 12:29:43] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 12:29:43,715 INFO: 127.0.0.1 - - [16/Jul/2024 12:29:43] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:04:56,698 INFO: Listado de inscripciones obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:04:56,703 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:04:56,716 INFO: 127.0.0.1 - - [16/Jul/2024 13:04:56] "GET /admin/inscripciones HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:04:56,751 INFO: 127.0.0.1 - - [16/Jul/2024 13:04:56] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:04:58,123 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:04:58,125 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:04:58,133 INFO: 127.0.0.1 - - [16/Jul/2024 13:04:58] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:04:58,156 INFO: 127.0.0.1 - - [16/Jul/2024 13:04:58] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:08,237 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:08,238 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:08,240 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:08] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:08,259 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:08] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:09,384 INFO: Listado de alumnos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:09,385 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:09,394 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:09] "GET /admin/alumnos HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:09,415 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:09] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:10,343 INFO: Listado de profesores obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:10,345 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:10,349 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:10] "GET /admin/profesores HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:10,369 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:10] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:11,310 INFO: Listado de clases obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:11,311 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:11,314 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:11] "GET /admin/clases HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:11,334 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:11] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:12,383 INFO: Listado de inscripciones obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:12,385 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:12,388 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:12] "GET /admin/inscripciones HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:12,408 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:12] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:13,165 INFO: Listado de descuentos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:13,166 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:13,175 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:13] "GET /admin/descuentos HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:13,193 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:13] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:14,869 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:14] "[32mGET /logout HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:14,875 INFO: Página principal cargada [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:06:14,876 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:06:14,881 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:14] "GET / HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:14,902 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:14] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:06:14,906 INFO: 127.0.0.1 - - [16/Jul/2024 13:06:14] "[36mGET /static/images/logo.png HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:10:21,281 INFO: 127.0.0.1 - - [16/Jul/2024 13:10:21] "GET /login HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:10:21,305 INFO: 127.0.0.1 - - [16/Jul/2024 13:10:21] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:11:10,762 WARNING: Intento fallido de inicio de sesión para usuario: Marprofesor [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:60]
2024-07-16 13:11:10,763 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:11:10,764 INFO: 127.0.0.1 - - [16/Jul/2024 13:11:10] "[31m[1mPOST /login HTTP/1.1[0m" 401 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:11:36,309 INFO: Inicio de sesión exitoso: Marprofesor [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:11:36,310 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:11:36,312 INFO: 127.0.0.1 - - [16/Jul/2024 13:11:36] "[32mPOST /login HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:11:36,318 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:11:36,319 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:11:36,320 INFO: 127.0.0.1 - - [16/Jul/2024 13:11:36] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:11:36,347 INFO: 127.0.0.1 - - [16/Jul/2024 13:11:36] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:13:50,043 INFO: Inicio de sesión exitoso: Marprofesor [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:13:50,045 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:13:50,046 INFO: 127.0.0.1 - - [16/Jul/2024 13:13:50] "[32mPOST /login HTTP/1.1[0m" 302 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:13:50,049 INFO: Dashboard cargado correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 13:13:50,050 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:13:50,053 INFO: 127.0.0.1 - - [16/Jul/2024 13:13:50] "GET /dashboard HTTP/1.1" 200 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:13:50,078 INFO: 127.0.0.1 - - [16/Jul/2024 13:13:50] "[36mGET /static/css/styles.css HTTP/1.1[0m" 304 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:13:52,848 WARNING: Intento de acceso a página de administrador sin permisos [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:60]
2024-07-16 13:13:52,851 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:13:52,853 INFO: 127.0.0.1 - - [16/Jul/2024 13:13:52] "[31m[1mGET /admin/clases HTTP/1.1[0m" 403 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:15:28,153 WARNING: Intento de acceso a página de administrador sin permisos [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:60]
2024-07-16 13:15:28,155 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:15:28,156 INFO: 127.0.0.1 - - [16/Jul/2024 13:15:28] "[31m[1mGET /admin/alumnos HTTP/1.1[0m" 403 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:15:42,707 WARNING: Intento de acceso a página de administrador sin permisos [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:60]
2024-07-16 13:15:42,709 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:15:42,710 INFO: 127.0.0.1 - - [16/Jul/2024 13:15:42] "[31m[1mGET /admin/profesores HTTP/1.1[0m" 403 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 13:16:40,418 WARNING: Intento de acceso a página de administrador sin permisos [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:60]
2024-07-16 13:16:40,419 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 13:16:40,419 INFO: 127.0.0.1 - - [16/Jul/2024 13:16:40] "[31m[1mGET /admin/profesores HTTP/1.1[0m" 403 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:12,984 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:12,985 INFO: [33mPress CTRL+C to quit[0m [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:12,989 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:13,374 WARNING:  * Debugger is active! [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:13,378 INFO:  * Debugger PIN: 733-530-386 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:15,443 INFO: Listado de alumnos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 17:57:15,462 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 17:57:15,471 ERROR: Error al obtener alumnos: 'alumno' is undefined [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:64]
2024-07-16 17:57:15,472 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 17:57:15,473 INFO: 127.0.0.1 - - [16/Jul/2024 17:57:15] "[35m[1mGET /admin/alumnos HTTP/1.1[0m" 500 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:57:17,944 INFO: Listado de alumnos obtenido correctamente [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:56]
2024-07-16 17:57:17,946 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 17:57:17,948 ERROR: Error al obtener alumnos: 'alumno' is undefined [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:64]
2024-07-16 17:57:17,949 ERROR: Error al guardar log en la base de datos: no existe la relación «logs»
LINE 2:             INSERT INTO logs (timestamp, level, message)
                                ^
 [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\logs.py:49]
2024-07-16 17:57:17,950 INFO: 127.0.0.1 - - [16/Jul/2024 17:57:17] "[35m[1mGET /admin/alumnos HTTP/1.1[0m" 500 - [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:58:08,988 INFO:  * Detected change in 'C:\\Users\\ibane\\Desktop\\proyectosiafactoria\\Grupo3_Musica\\app.py', reloading [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
2024-07-16 17:58:09,021 INFO:  * Restarting with stat [in C:\Users\ibane\Desktop\proyectosiafactoria\Grupo3_Musica\venv\Lib\site-packages\werkzeug\_internal.py:97]
